checkov/arm/runner.py

```diff
@@ -304,5 +306,12 @@ def build_record(
             file_abs_path=file_abs_path,
             severity=check.severity,
         )
+        if self.breadcrumbs:
+            breadcrumb = self.breadcrumbs.get(record.file_path, {}).get(record.resource)
+            if breadcrumb:
+                record = GraphRecord(record, breadcrumb)
         record.set_guideline(check.guideline)
         report.add_record(record=record)
+
+    def extract_file_path_from_abs_path(self, path: Path) -> str:
+        return f"/{os.path.relpath(path, self.root_folder)}"
```

Comment:
this will not work on all OS, use `os.path.sep` instead of `/`.

Reply:
Done